website_multi_theme/models/ir_ui_view.py

```diff
@@ -13,12 +15,35 @@
 class IrUiView(models.Model):
     _inherit = 'ir.ui.view'
 
-    multi_theme_generated = fields.Boolean(
-        name="Generated from multi theme module",
-        readonly=True,
-        help="Was this view autogenerated by website_multi_theme?",
-    )
     was_active = fields.Boolean(
+        readonly=True,
         help="Indicates if the view was originally active before converting "
              "the single website theme that owns it to multi website mode.",
     )
+    origin_view_id = fields.Many2one(
+        "ir.ui.view",
+        string="Copied from",
+        readonly=True,
+        help="View from where this one was copied for multi-website"
+    )
+    copy_ids = fields.One2many(
+        "ir.ui.view",
+        "origin_view_id",
+        string="Copies",
+        readonly=True,
+        help="Duplicates of this view"
+    )
+
+    @api.model
+    def _customize_template_get_views(self, key, full=False, bundles=False):
+        """This method is used to prepare items
+           in 'Customize' menu of website Editor"""
+        views = super(IrUiView, self)._customize_template_get_views(
+            key, full=full, bundles=bundles
+        )
+
```

Comment:
These empty lines don't add anything useful and PEP8 discourages its use. Better to strip them.